src/ops/diff/verify/tests.rs

```diff
@@ -1496,96 +1485,235 @@ fn test_try_apply_diff_with_deleted_db_dependency() {
     log::info!("second trie dropped")
 }
 
-use quickcheck::{Arbitrary, Gen, QuickCheck, TestResult};
+use quickcheck::{Gen, QuickCheck, TestResult};
 
-fn fixed_key_unique_random_data(values: HashSet<RandomFixedData>) -> debug::EntriesHex {
-    let mut g = Gen::new(RNG_DATA_SIZE);
-    let mut entries = vec![];
-    for val in values.into_iter() {
-        let key = FixedKey::arbitrary(&mut g);
-        let entry = (key.0.to_vec(), Some(val.0.to_vec()));
-        entries.push(entry);
-    }
+fn reverse_changes(changes: Vec<Change>) -> Vec<Change> {
+    changes
+        .into_iter()
+        .map(|i| match i {
+            Change::Insert(h, d) => Change::Removal(h, d),
+            Change::Removal(h, d) => Change::Insert(h, d),
+        })
+        .collect()
+}
 
-    debug::EntriesHex::new(entries)
+trait ChildExtractor: Serialize {
+    type Child: ChildExtractor;
+    fn extract(data: &[u8]) -> Vec<H256> {
+        crate::debug::no_childs(data)
+    }
+    // Change existing data, so it will refer to link root
+    fn merge_root_to_data(data: &[u8], _root: H256) -> Vec<u8> {
```

Comment:
this trait method is redundant. stuff can probably simplified a bit, if it's removed